### PR TITLE
feat: expose eth_getProof api

### PR DESCRIPTION
**Pull request by NazariiDenha**

### CODE REVIEW

File: rpc/eth_api.go [hunk 1]
```diff
@@ -539,6 +539,14 @@ func (api *ethAPI) FeeHistory(
 	return w3c.Eth.FeeHistory(blockCount, lastBlock, rewardPercentiles)
 }
 
+// GetBalance returns the amount of wei for the given address in the state of the
+// given block number.
+func (api *ethAPI) GetProof(ctx context.Context, addr common.Address, storageKeys []string, blockNrOrHash web3Types.BlockNumberOrHash) (*web3Types.AccountResult, error) {
+	w3c := GetEthClientFromContext(ctx)
+	api.inputBlockMetric.Update2(&blockNrOrHash, "eth_getProof", w3c.Eth)
```
Comment (Thegaram) [inline rpc/eth_api.go:546]: What's the difference between `Update1` and `Update2`?

Comment (NazariiDenha) [reply rpc/eth_api.go:546]: `Update1` accepts `BlockNumber`, `Update2` accepts `BlockNumberOrHash`

**Review by Thegaram — APPROVED** [review]

**Comment by OrestTa** [comment]
@NazariiDenha @Thegaram What's the deployment plan, after staging? (Makoto/ENS is trying to use https://hackmd.io/@Eec99y4eQJm9h5ixr0T4FQ/rkhaPJyxh )

**Comment by Thegaram** [comment]
Already deployed on alpha.

**Pull request merged**